README.md

```diff
@@ -10,13 +10,22 @@ This package enables Firebase OAuth on desktop via webview
 
 ## Installation
 
-Add dependency
+### Linux setup
+
+To display webview on Linux, `webkit2gtk-4.0` is used, if you don't have it already installed:
+```bash
+apt install webkit2gtk-4.0
```

Comment:
how would that work in a distributed app?

Reply:
I think it will be compiled within the final bundle. But not sure, will double check

Reply:
I uninstalled `webkit2gtk-4.0-dev` after building a release version, I was able to run the bundle but not compile in debug. So yeah it's just a development step.